packages/blade/src/components/Accordion/types.ts

```diff
@@ -0,0 +1,13 @@
+import type { IconComponent } from '~components/Icons';
+
+type StyledAccordionButtonProps = {
+  isExpanded: boolean;
+};
+
+type AccordionButtonProps = {
+  index?: number;
+  icon?: IconComponent;
+  children: string;
+};
+
+export { StyledAccordionButtonProps, AccordionButtonProps };
```

Comment:
how are these both diff? is StyledAccordionButton more like an AccordionPanel?

Reply:
Bit different, here's the breakdown I've used:

- AccordionItem (which users use) -> renders a Collapsible
  - AccordionButton -> used for Collapsible trigger (web has different markup requirements for a11y)
    - StyledAccordionButton -> for styling, animations, hover, focus, etc.
      - web: css based styling, animations
      - native: will be Pressable + Animated

Once I implement native I'll abstract the common things out from web modules